packages/appstore/src/components/modals/real-wallets-upgrade/components/ready-to-upgrade-wallets/test/ready-to-upgrade-wallets.spec.tsx

```diff
@@ -3,33 +3,46 @@ import { render, screen } from '@testing-library/react';
 import ReadyToUpgradeWallets from '../ready-to-upgrade-wallets';
 import { StoreProvider, mockStore } from '@deriv/stores';
 
-let mock = mockStore({});
-
-const checkContainerReadyToUpgradeWallets = () => {
```

Comment:
@sergei-deriv let's keep this component for rendering the wrapper and just check different assertions in each test

const  coontainerReadyToUpgradeWallets = () => {
     const toggleCheckbox = jest.fn();
    const wrapper = ({ children }: { children: JSX.Element }) => <StoreProvider store={mock}>{children}</StoreProvider>;

    const { container } = render(<ReadyToUpgradeWallets is_eu={false} toggleCheckbox={toggleCheckbox} />, {
        wrapper,
        });
  }